x-pack/plugins/search_inference_endpoints/public/components/all_inference_endpoints/render_table_columns/render_actions/actions/delete/confirm_delete_endpoint/confirm_delete_endpoint.scss

```diff
@@ -0,0 +1,3 @@
+.confirmDeleteInferenceModal {
+  width: 600px;
```

Comment:
We should not be hardcoding pixel values. If hardcoding this with is really necessary (and I don't think it is) we should be using EUI variables. More importantly: for an EuiModal width you should just be setting the `width` property, not using CSS.

Reply:
Updated. 

@mdefazio Do we require to set the modal width to 600px? I am trying to set the width according to [design](https://www.figma.com/design/duLXTmppjDWJyufbr5jyLW/Inference-Endpoint-Management-[Search-170]?node-id=2936-226089&t=5MOFGDalECUvaSzh-0).

Reply:
Sorry @saikatsarkar056 a slight tweak from what we just talked about.  We shouldn't even need the `maxWidth` as the modal defaults to using the right maxWidth per breakpoint. 


[EUI Docs - Modals](https://eui.elastic.co/#/layout/modal)

Reply:
Updated. Thank you.